### DISCUSSION TITLE
Make chai-json-schema available by default in Unit Tests

### DISCUSSION THREAD

**Discussion by x80486** [Kong/insomnia#6268] [Ideas] [answered]
It's common to check if a given response follows certain JSON schema. Currently it's not possible to do this in a straightforward way, so it would be better if [chai-json-schema](https://www.chaijs.com/plugins/chai-json-schema/) could be included by default.

**Comment by kobenguyent** [comment, marked as answer]
Fixing here https://github.com/Kong/insomnia/pull/6532

**Comment by subnetmarco** [comment]
We are prioritizing this issue. We need to test that it works in our CLI before shipping, ideally in Insomnia v8.5 in a couple of weeks.
